src/CarbonAware.DataSources/CarbonAware.DataSources.Registration/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -12,9 +13,7 @@ public static class ServiceCollectionExtensions
 {
     public static IServiceCollection AddDataSourceService(this IServiceCollection services, IConfiguration configuration)
     {
-        var dataSources = configuration.GetSection(DataSourcesConfiguration.Key).Get<DataSourcesConfiguration>() ?? new DataSourcesConfiguration();
```

Comment:
What was the reason behind this change? Feels a bit out of place with this PR, was it an issue of not pulling in the latest datasource config changes?